packages/core/content-manager/admin/src/pages/EditView/components/InputRenderer.tsx

```diff
@@ -85,13 +96,15 @@ const InputRenderer = ({ visible, hint: providedHint, ...props }: InputRendererP
       // @ts-expect-error – TODO: fix this type error in the useLazyComponents hook.
       return <CustomInput {...props} hint={hint} disabled={fieldIsDisabled} />;
     } else {
-      <FormInputRenderer
-        {...props}
-        hint={hint}
-        // @ts-expect-error – this workaround lets us display that the custom field is missing.
-        type={props.attribute.customField}
-        disabled={fieldIsDisabled}
-      />;
+      return (
```

Comment:
ℹ️ This should be returned right...?

Reply:
yup looks like that was a mistake. Also you probably don't need the else since the first clause has a return